ex_standards/ex_http_status.md

```diff
@@ -8,35 +8,35 @@ Some might ok with 2 options, choose one and justify it.
 
 ----
 
-* I cannot access that web if I'm not logged:
-  *  
+* I cannot access that web if I'm not logged: 
+  *  401
 
-* I'm logged but I cannot access that page
-  * 
+* I'm logged but I cannot access that page 
+  * 403
 
-* The item with that Id was deleted
-  * 
+* The item with that Id was deleted 
+  * 404
 
-* I started uploading, but I have to check another link to see if it is there
-  * 
+* I started uploading, but I have to check another link to see if it is there 
+  * 307
 
-* That query parameter is wrong
-  * 
+* That query parameter is wrong 
+  * 400
 
-* There are no resources with that ID
-  * 
+* There are no resources with that ID 
+  * 204
```

Comment:
sería un erro 404 (not found), 204 significa "OK pero non che devolvo nada no corpo da resposta"